### PR TITLE
DTSPO-4182-image-automation-changes-for-aat

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by linusnorton — DISMISSED** [review]

**Review by rishikrsharma — DISMISSED** [review]

**Comment by tomaszpowroznik** [comment]
family-public-law has not been configured to support flux 2

**Review by jakub-kozlowski — DISMISSED** [review]

**Review by dharmendrak — APPROVED** [review]

**Review by btensay — APPROVED** [review]

**Review by YasharRahvar — DISMISSED** [review]

**Review by verityit — APPROVED** [review]
Idam entries look good

approval based on idam only, as requested by Avi

**Review by davejones74 — DISMISSED** [review]

**Review by apereverzin — DISMISSED** [review]

**Review by mkayad — DISMISSED** [review]
LGTM

**Review by btensay — DISMISSED** [review]

**Review by jthmcts — DISMISSED** [review]

**Review by mdayican — DISMISSED** [review]

**Review by gauravchawla73463 — DISMISSED** [review]

File: k8s/aat/common/idam/idam-api.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
     version: 0.3.9
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:prod-97407f6-20210803165122
+      image: hmctspublic.azurecr.io/idam/api:prod-97407f6-20210803165122   #{"$imagepolicy": "flux-system:idam-api"}
```
Comment (adusumillipraveen) [inline k8s/aat/common/idam/idam-api.yaml:21]: Lets remove this as IDAM is usually not automated on aat.

**Review by thomast1906 — APPROVED** [review]

**Review by lewisbirks — DISMISSED** [review]
Approved based on no changes for family public law

**Review by linusnorton — APPROVED** [review]

**Review by gauravchawla73463 — APPROVED** [review]

**Review by rishikrsharma — DISMISSED** [review]

**Review by mdayican — DISMISSED** [review]

**Review by jthmcts — APPROVED** [review]

**Review by mkayad — DISMISSED** [review]
LGTM

**Review by david-gimelle — DISMISSED** [review]

**Pull request merged**